ui/app/pages/send/send.component.js

```diff
@@ -187,6 +193,25 @@ export default class SendTransactionScreen extends Component {
         this.updateGas();
       }
     }
+
+    // If selecting ETH after selecting a token, clear token related messages.
+    if (prevSendToken && !sendToken) {
```

Comment:
I noticed a bug here with the "Known Contract Address" warning that also presented itself for the new error. It seems these messages are not intended to show when ETH is the selected asset, however, when switching back to ETH from an ERC-20 token, the messages don't clear properly. The `validate` method of this component does not trigger when selecting a new send asset